be/src/formats/parquet/column_reader.h

```diff
@@ -55,12 +47,20 @@ class ColumnReader {
 
     virtual ~ColumnReader() = default;
 
-    virtual Status prepare_batch(size_t* num_records, ColumnContentType content_type, Column* column) = 0;
+    // Will return actually rows read or InternalError
+    // If meet EOF, will return 0
```

Comment:
It's great to comment with special cases of this function, but better to add comment: "what this function is for".